test_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,91 @@
+*****************************
+Quick-start development guide
+*****************************
+
+This guide describes all steps needed to test another author's pull request.
+
+.. contents:: Topics
+
+Prepare your environment
+========================
+
+We assume that you use Linux as a work environment (you can use a virtual machine as well) and have ``git`` installed.
+
+1. If possible, make sure that you have installed and started ``docker``. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements, and tests are running properly isolated and in the exact same environments as in CI. You often can also use ``podman`` with the ``docker`` executable shim, so if you have that you probably do not need to install ``docker``.
+
+2. `Install <https://docs.ansible.com/ansible/devel/installation_guide/intro_installation.html>`_ Ansible or ansible-core.
+
+3. Create the following directories in your home directory:
+
+.. code:: bash
+
+  mkdir -p ~/ansible_collections/NAMESPACE/COLLECTION_NAME
+
+For example, if the collection is ``community.general``, it will be:
+
+.. code:: bash
+
+  mkdir -p ~/ansible_collections/community/general
+
+If the collection is ``ansible.posix``, it will be:
```

Comment:
```suggestion
If the collection is ``ansible.posix``, it will be:
```
it's a good idea to provide this additional example. Maybe add the same to `create_pr_quick_start_guide.rst`?

Reply:
I've copied it from `create_pr_quick_start_guide.rst`, so it's already there. Or do you mean adding this somewhere else?

Reply:
ah, no then:) I forgot that it's there